### ISSUE TITLE
Maint-1.2: make install jquery-plugins fails

### ISSUE THREAD

**Issue by rthieledesy** [inveniosoftware/invenio#3644]
While the installation process, the install-jquery-plugins fails:

```
--2016-06-30 09:03:32--  http://jquery-multifile-plugin.googlecode.com/svn-history/r54/trunk/jquery.MultiFile.pack.js
Resolving jquery-multifile-plugin.googlecode.com... 74.125.136.82, 2a00:1450:4013:c01::52
Connecting to jquery-multifile-plugin.googlecode.com|74.125.136.82|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2016-06-30 09:03:32 ERROR 404: Not Found.

make: *** [install-jquery-plugins] Error 8
```

The following packages can not load automatically.

**Comment by egabancho** [comment]
I believe this was fixed in https://github.com/inveniosoftware/invenio/commit/d8fc81d9b0a5082e0156d0e34464ccb6df026fdc
can you please double check?

**Comment by rthieledesy** [comment]
I will make this.
I was testing this for tag v1.1.6 and tag v1.2.1. In both cases is the behaviour like the above description.

**Comment by rthieledesy** [comment]
Thanks, the maint-1.2 is ok.

**Comment by tiborsimko** [comment]
Yes, this issue was fixed in both 1.1 and 1.2, but the `maint-1.x` branches weren't released yet.
